keyboards/converter/sun_usb/README

```diff
@@ -0,0 +1,79 @@
+Sun to USB keyboard protocol converter
```

Comment:
Also, could you rename this to `readme.md` and make sure that the file is lowercase? 

And reformat the readme so that it conforms more to this: 
https://docs.qmk.fm/#/documentation_templates?id=keyboard-readmemd-template

It doesn't have to be like that exactly, but mostly the header stuff (eg the `====` stuff).